tests/test_stats_verifier.py

```
@@ -15,41 +485,29 @@ def test_stats_verification():
-        'file': ('sales.csv', csv_content, 'text/csv')
-    }
-    data = {
-        'query': query,
-        'provider': 'azure_openai'
-    }
-    
+    files = {'file': ('sales.csv', csv_content, 'text/csv')}
+    data = {'query': query, 'provider': 'azure_openai'}
+
     try:
         response = requests.post(f"{BASE_URL}/verify/stats", files=files, data=data)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)